www/src/components/docs/outdatedDocsBanner.astro

```diff
@@ -52,7 +52,7 @@ Astro.response.headers.set("Cache-Control", `s-max-age=${ONE_WEEK}`);
   diffInDays > 0 && (
     <div
       dir="ltr"
-      class="mx-4 mb-4 rounded-lg bg-t3-purple-200/50 p-4 font-medium dark:bg-t3-purple-200/10 lg:w-full"
+      class="mx-4 mb-6 rounded-lg bg-t3-purple-200/50 p-4 font-medium dark:bg-t3-purple-200/10 lg:mx-0 lg:ml-4"
```

Comment:
what is the reason for the mb change?

Reply:
Sorry, should have mentioned this in the PR description. It's for consistent spacing. Notice the gap between the banner and breadcrumbs.

- Before

![Screenshot_20221211-050306_Brave.png](https://user-images.githubusercontent.com/89210438/206879549-17250d31-9bed-451b-a09b-813ee82061eb.png)

- After

![Screenshot_20221211-050527_Brave.png](https://user-images.githubusercontent.com/89210438/206879554-9257d21c-6c6d-4a06-9c10-8b589dba62c6.png)